src/commands/project/convert/mdapi.ts

```diff
@@ -66,19 +66,19 @@ export class Mdapi extends SfCommand<ConvertMdapiJson> {
       summary: messages.getMessage('flags.manifest'),
       exists: true,
     }),
-    'metadata-path': arrayWithDeprecation({
+    'metadata-dir': arrayWithDeprecation({
       char: 'p',
       aliases: ['metadatapath'],
       deprecateAliases: true,
-      description: messages.getMessage('flagsLong.metadata-path'),
-      summary: messages.getMessage('flags.metadata-path'),
+      description: messages.getMessage('flagsLong.metadata-dir'),
+      summary: messages.getMessage('flags.metadata-dir'),
       exclusive: ['manifest', 'metadata'],
     }),
     metadata: arrayWithDeprecation({
       char: 'm',
       description: messages.getMessage('flagsLong.metadata'),
       summary: messages.getMessage('flags.metadata'),
-      exclusive: ['manifest', 'metadatapath'],
```

Comment:
we have an eslint rule to prevent duplication, but not one that checks that these are actually flags.  I suppose that'll be harder with `extends x` commands where the flags are at various levels of inheritance but still doable.

[W-12740580](https://gus.my.salesforce.com/a07EE00001No6xTYAR)